manifests/params.pp

```diff
@@ -9,7 +9,7 @@
       $root_group    = 'wheel'
       $logrotate_bin = '/usr/local/sbin/logrotate'
       $conf_params = {
```

Comment:
@mookie- Can it just become `$conf_params = {}` ? (like in the `default` case)

Reply:
@alexjfisher I thought about that too but it was added with a commit called "Repair tests" (https://github.com/mookie-/puppet-logrotate/commit/13ce41d647e35ba618f114b4e131303ad0e5117f#diff-2f383a197b1a428561b364755701d60eR12)
To be honest I haven't looked at the tests yet if it's important to be there. I'll check that tomorrow.

Reply:
No worries.  The current code is broken and this is a definite improvement.